### DISCUSSION TITLE
Crash on launch

### DISCUSSION THREAD

**Discussion by Tristen895** [DFHack/dfhack#2524] [Q&A]
I dropped the files directly into the game bin but it crashes on launch, is there anything I can try to fix it? I made sure to replace the sdl files, and did a fresh install and tried again no luck.

**Comment by ab9rf** [comment]
could you provide more information? which version of dfhack and of df is this in relation to?

**Reply by Tristen895** [reply]
game version 50.04 and dfhack latest version windos x64

**Reply by lethosor** [reply]
What version number of DFHack?

**Reply by Tristen895** [reply]
Processor AMD Ryzen 5 3600
GPU          Radeon RX 5500 XT
RAM         Ballistix ddr4 8gb x2 3000mhz

idk if it could be system related that's my main specs I think I saw something about dfhack possible having issues with certain gpus I'm not sure I may just try on my older pc just to check at some point. assuming the latest game version is compatible with dfhacks release rn
that is.

**Reply by Tristen895** [reply]
sorry didnt see your reply cuz i was typing one sec

**Reply by Tristen895** [reply]
[dfhack-0.47.05-r8-Windows-64bit.zip](https://github.com/DFHack/dfhack/releases/download/0.47.05-r8/dfhack-0.47.05-r8-Windows-64bit.zip)

**Reply by Tristen895** [reply]
didn't mean to copy the file XD but that one

**Reply by Tristen895** [reply]
idk if its on any other launcher but I am on steam aswell.

**Reply by Tristen895** [reply]
Oh and idk if it would make a difference but its installed on a secondary hdd. any help is appreciated, thank you.

**Reply by myk002** [reply]
DFHack versions are tied to specific DF versions. You have the DFHack version for DF 0.47.05 installed, which will not work with DF 50.X. You can pick up development builds for DFHack 50.04 at dfhack.org/builds but don't expect anything to work yet. We still have quite a bit to fix before it's basically functional.

**Reply by Tristen895** [reply]
ok that's kind what I expected to hear, thank you for clarifying! cant wait to test it out when I can, keep up the great work :)

**Reply by Tristen895** [reply]
I just realized of course your released versions correspond to the version of the game I coulda answered this myself ; ;

sorry to waste everyone's time "im a ditz" and I hope you all had a great holiday :x
